src/convertToObject.js

```diff
@@ -7,6 +7,24 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const obj = {};
+
+  const strings = sourceString.split(';').filter(Boolean);
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes the code much more readable.